views/file/addserver.html

```diff
@@ -0,0 +1,36 @@
+<div class="flex-fill w-100 d-flex flex-column">
+  <div class="modal-body container flex-fill">
+    <form>
+      <div class="form-group m-0" ng-if="!ctrl.loading">
+        <label for="serverType">Select a server type:</label>
+        <select title="ServerType" class="custom-select w-50"
+            ng-model="ctrl.serverType"
+            ng-change="ctrl.onServerTypeChange_()"
+            ng-options="value.label for value in ctrl.items"
+            ng-disabled="ctrl.loading">
+        </select>
+      </div>
+    </form>
+    <div>
+      <uiswitch items="ctrl.serverType" directive-function="ctrl.getUi" scope-update-function="ctrl.updateUiScope"></uiswitch>
+      <div class="alert alert-danger" ng-if="error">
```

Comment:
This error block and the loading block below are no longer used and can be removed.